app/components/appMainModule/localityPageModule/state.js

```diff
@@ -16,7 +16,11 @@ module.exports = function($stateProvider) {
       template: '<locality-listing locality="locality"></locality-listing>',
       resolve: {
         locality: function($stateParams, disclosureApi) {
-          return disclosureApi.locations.get({locality_id: $stateParams.locality_id});
+          return disclosureApi.locality.current_ballot({
+            locality_id: $stateParams.locality_id
+          }).then(function(ballot) {
+            disclosureApi.ballot.summary({ballot_id: ballot.ballot_id});
+          });
```

Comment:
@adborden This is more complex than before; I like the way this API is defined, but not sure if this is the right way to chain API calls.